flixel/system/frontEnds/CameraFrontEnd.hx

```diff
@@ -57,22 +57,22 @@ class CameraFrontEnd
 	 * Handy for PiP, split-screen, etc.
 	 * @see flixel.FlxBasic.cameras
 	 *
-	 * @param	NewCamera         The camera you want to add.
-	 * @param	DefaultDrawTarget Whether to add the camera to the list of default draw targets. If false, 
+	 * @param	newCamera         The camera you want to add.
+	 * @param	defaultDrawTarget Whether to add the camera to the list of default draw targets. If false, 
 	 *                            `FlxBasics` will not render to it unless you add it to their `cameras` list.
 	 * @return	This FlxCamera instance.
 	 */
-	public function add<T:FlxCamera>(NewCamera:T, DefaultDrawTarget:Bool = true):T
+	public function add<T:FlxCamera>(newCamera:T, defaultDrawTarget:Bool = true):T
 	{
-		FlxG.game.addChildAt(NewCamera.flashSprite, FlxG.game.getChildIndex(FlxG.game._inputContainer));
-		
-		list.push(NewCamera);
-		if (DefaultDrawTarget)
-			defaults.push(NewCamera);
+		FlxG.game._cameraContainer.addChild(newCamera.flashSprite);
+
+		list.push(newCamera);
+		if (defaultDrawTarget)
+			defaults.push(newCamera);
```

Comment:
Why the change from `addChildAt` to `addChild`? wouldn't this add new cameras to show behind old ones, rather than in front?